### ISSUE TITLE
Documentation regarding config mode is hard to find.

### ISSUE THREAD

**Issue by jtokash** [mcer12/Hugh-ESP8266#10]
Please put a link to https://github.com/mcer12/Hugo-ESP8266/wiki/Sketch:-HTTP-(Basic-URL-trigger) in the main page of the wiki.

**Comment by mcer12** [comment]
Hi @jtokash I don't quite get what you mean. All wiki sections are on the main page of the wiki as well as on project's home page. What changes do you propose?

**Comment by jtokash** [comment]
Sorry for the confusion.  I clicked through each of the links and didn't get what I needed,  so I didn't realize you were trying to link to it.  Turns out the link you have is just typo'd.

On the main page of the wiki, there is a link to  Sketch:-HTTP-(Basic-URL), which does not exist.  The page you want to link to is Sketch:-HTTP-(Basic-URL-trigger).

[Edit] I do now see the list of pages in the sidebar is different than the list of sections at the top of the page, but didn't realize it at the time.  :-)
